test/foundry/BlastYield.t.sol

```diff
@@ -0,0 +1,98 @@
+        address _blastPoints,
+        address _blastPointsOperator,
+        address _owner,
+        address _usdb,
+        address _weth
+    ) BlastYield(_blast, _blastPoints, _blastPointsOperator, _owner, _usdb, _weth) OwnableTwoSteps(_owner) {}
+
+    function claim(address wethReceiver, address usdbReceiver) public onlyOwner {
+        _claim(wethReceiver, usdbReceiver);
+    }
+}
+
+contract BlastYieldOwnableTwoSteps_Test is Test {
+    MockWETH private weth;
+    MockERC20 private usdb;
+    MockYield private mockYield;
+    MockPoints private mockPoints;
+    BlastYieldOwnableTwoSteps private blastYieldOwnableTwoSteps;
+
+    address public owner = address(69);
+    address public operator = address(420);
+    address public user1 = address(1);
+    address private constant TREASURY = address(69420);
+    address internal constant BLAST_POINTS = 0x2536FE9ab3F511540F2f9e2eC2A805005C3Dd800;
+    bytes32 private constant OPERATOR_ROLE = keccak256("OPERATOR_ROLE");
+
+    function setUp() public {
+        vm.etch(BLAST_POINTS, address(new MockPoints()).code);
+        weth = new MockWETH();
+        usdb = new MockERC20("USDB", "USDB");
+        mockYield = new MockYield();
+        blastYieldOwnableTwoSteps = new BlastYieldOwnableTwoSteps(
+            address(mockYield),
+            BLAST_POINTS,
+            operator,
+            owner,
+            address(usdb),
+            address(weth)
+        );
+    }
+
+    function test_setUpState() public {
+        assertEq(blastYieldOwnableTwoSteps.WETH(), address(weth));
+        assertEq(blastYieldOwnableTwoSteps.USDB(), address(usdb));
+
+        (IBlast__YieldMode yieldMode, IBlast__GasMode gasMode, address governor) = mockYield.config(
+            address(blastYieldOwnableTwoSteps)
+        );
+        assertEq(uint8(yieldMode), uint8(IBlast__YieldMode.CLAIMABLE));
+        assertEq(uint8(gasMode), uint8(IBlast__GasMode.CLAIMABLE));
+        assertEq(governor, owner);
+
+        IERC20Rebasing__YieldMode wethYieldMode = weth.yieldMode(address(blastYieldOwnableTwoSteps));
+        assertEq(uint8(wethYieldMode), uint8(IERC20Rebasing__YieldMode.CLAIMABLE));
+
+        IERC20Rebasing__YieldMode usdbYieldMode = usdb.yieldMode(address(blastYieldOwnableTwoSteps));
+        assertEq(uint8(usdbYieldMode), uint8(IERC20Rebasing__YieldMode.CLAIMABLE));
+    }
+
+    function test_claim() public asPrankedUser(owner) {
+        blastYieldOwnableTwoSteps.claim(TREASURY, TREASURY);
+
+        assertEq(weth.balanceOf(address(blastYieldOwnableTwoSteps)), 0);
+        assertEq(usdb.balanceOf(address(blastYieldOwnableTwoSteps)), 0);
+        assertEq(weth.balanceOf(TREASURY), 1 ether);
+        assertEq(usdb.balanceOf(TREASURY), 1 ether);
+    }
+
+    function test_claim_RevertIf_NotOwner() public asPrankedUser(user1) {
+        vm.expectRevert(IOwnableTwoSteps.NotOwner.selector);
+        blastYieldOwnableTwoSteps.claim(TREASURY, TREASURY);
+    }
+
+    modifier asPrankedUser(address user) {
```

Comment:
asPrankedUser is a foundry function, why do we need to define it?

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/591cf53b7463bd1c9e1145a176b539f58f4b24c2